src/partners.html

```diff
@@ -0,0 +1,174 @@
+---
+layout: default
+permalink: /partners
+title: Partners
+---
+
+{% include clipped-header.html %} {% include clipped-stats.html %}
+
+<div class="row justify-content-center">
+  <div class="clipped-header background-purple-4 col-11 col-md-10 mt-4 mt-md-5 mb-4 py-4 py-md-5 px-4 px-md-0">
+    <div class="offset-md-2 col-md-8 py-3 py-md-5 px-2 px-md-2">
+      <h1 class="text-white mb-3">Partners</h1>
+      <p class="text-white mb-3">
+        Bringing a modern and consistent transportation experience to all Californians requires help from a broad range of
+        experts. That's why Cal-ITP is proud to work collaboratively with our partners.
+      </p>
+    </div>
+  </div>
+</div>
+
+<section class="row justify-content-center">
+  <div class="col-md-10 col-11 px-0 mx-0">
+    <div class="offset-md-2 col-md-8 mt-md-4 mt-2">
+      <section class="pb-md-2">
+        <img class="w-100 rounded-1" src="/images/partners-cdt-background.png" />
+        <h2 class="my-md-4 my-3">California Department of Technology</h2>
+        <p>
+          We collaborate with the
+          <a class="link-body-emphasis fw-normal" href="https://cdt.ca.gov/" target="_blank"
+            >California Department of Technology</a
+          >
+          (CDT) to provide transit riders with a seamless way to qualify for and enroll in transit benefits online.
+        </p>
+        <figure class="quote my-md-4 my-3">
```

Comment:
Vertical spacing between elements should be 24px Desktop / 16px Mobile according to Figma.

The preceding `<p>` tag has a 16px bottom margin applied, so when the top margin is added to this `<figure>` the spacing goes beyond 24px.

Recommend:

* Remove `<figure>` top-margin for Mobile
* Only add 8px `<figure>` top margin for Desktop

Reply:
The preceding `<p>` tag's 16px bottom margin actually doesn't add to the `<figure>`s top margin. The yellow area below is 16px: 


<img width="1512" alt="image" src="https://github.com/user-attachments/assets/faf9351b-33cc-4d80-87d0-55118d89c4f0">

Unless maybe this is a Linux difference (???)

Reply:
Just checked on Chrome + Linux and it has the same behavior as Firefox/Chrome + Mac does for me -- the margins aren't cumulative.

Reply:
Yeah, you are right -- they both have margin in that space, but they aren't being added together. Thanks!

Reply:
Yeah it's confusing!!!!